resolwe_bio/tests/unit/test_management_commands.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=missing-docstring
+from django.core.management import call_command
+
+from resolwe.flow.models import Data, Storage
+from resolwe_bio.utils.test import BioProcessTestCase
+from resolwe_bio.models import Sample
+
+
+class GenerateSamplesTest(BioProcessTestCase):
+
+    def test_generate_samples(self):
+        call_command('generate_samples', '-s=1', '-p=0', '--rseed')
+        sample = Sample.objects.last()
+        if sample:
+            sample_data = sample.data.all().order_by('id')
```

Comment:
This test was failing because objects from `sample.data.all()` weren't ordered by `id` and checks bellow didn't pass. Is it possible that objects should be ordered by `id` by default and test failed because of something else?